test/parallel/test-npm-install.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
```

Comment:
I don't see any `crypto`/`tls`/`https` usage in this test, or is it because npm uses an `https`-based registry by default?